File: corporate-reporting/src/types/FinancialRiskRiskMitigationAction.ts
import { IFinancialRiskMitigationAction } from "./FinancialRiskMitigationAction";
import { IFinancialRisk } from "./FinancialRisk";
import { IRiskRiskMitigationAction, RiskRiskMitigationAction } from "./RiskRiskMitigationAction";

export interface IFinancialRiskRiskMitigationAction extends IRiskRiskMitigationAction {
    Risk?: IFinancialRisk;
    RiskMitigationAction?: IFinancialRiskMitigationAction;
}

export class FinancialRiskRiskMitigationAction extends RiskRiskMitigationAction implements IFinancialRiskRiskMitigationAction { }

File: ORB.Core/Models/FinancialRiskRiskMitigationAction.cs
using System;
using System.Collections.Generic;

namespace ORB.Core.Models
{
    public partial class FinancialRiskRiskMitigationAction : RiskRiskMitigationAction
    {
        public virtual FinancialRisk Risk { get; set; }
        public virtual FinancialRiskMitigationAction RiskMitigationAction { get; set; }
    }
}
